src/sidebar/components/Annotation/AnnotationEditor.tsx

```diff
@@ -175,7 +176,7 @@ function AnnotationEditor({
     >
       <MarkdownEditor
         textStyle={textStyle}
-        label="Annotation body"
+        label={`Enter ${isReplyAnno ? 'reply' : 'comment'}`}
```

Comment:
I would suggest constructing each of the strings separately. When we eventually get around to translating the interface, we will need to translate each of the two strings separately.